assets/css/refills/_tooltip.scss

```diff
@@ -0,0 +1,65 @@
+.tooltip-item {
+  $base-border-color: gainsboro !default;
```

Comment:
Color `gainsboro` should be written in hexadecimal form as `#dcdcdc`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#colorkeyword)